src/Stripe.jsx

```diff
@@ -9,19 +10,35 @@ class Stripe extends React.Component {
 		const {
 			children,
 			className,
+			backgroundImage,
+			collection,
+			inverted,
+			isHero,
 			...other
 		} = this.props;
 
 		const classNames = cx(
 			'stripe',
+			{
+				'stripe--collection': collection,
+				'stripe--inverted inverted': inverted,
+				'stripe--hero inverted': typeof backgroundImage === 'string' || isHero
+			},
 			className
 		);
 
 		return (
 			<div
 				className={classNames}
+				style={backgroundImage ? {backgroundImage: `url(${backgroundImage})`} : null}
 				{...other}>
-					{children}
+				{ isHero ?
+					<Bounds className='stripe-heroContent'>
+						{children}
+					</Bounds>
+					:
```

Comment:
sorry, the colon

```jsx
</Bounds> :
```

instead of

```jsx
</Bounds>
:
```

Stylistically, it's a little unconventional to have a colon alone on a line